docs/high-performance/MemoryOwner.md

```diff
@@ -0,0 +1,72 @@
+---
+title: MemoryOwner&lt;T>
+author: Sergio0694
+description: A buffer type implementing `IMemoryOwner<T>` that rents memory from a shared pool
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, parallel, high performance, net core, net standard
+dev_langs:
+  - csharp
+---
+
+# MemoryOwner&lt;T>
+
+The [MemoryOwner&lt;T>](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.highperformance.buffers.memoryowner-1) is a buffer type implementing `IMemoryOwner<T>`, an embedded length property and a series of performance oriented APIs. It is essentially a lightweight wrapper around the `ArrayPool<T>` type, with some additional helper utilities.
+
```

Comment:
Could be useful here to have more info on the types of scenarios where this type of helper would be useful. e.g. allocating bytes for image processing or something else...

Reply:
Done in https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/pull/301/commits/1a57734bd1d853d4b44364a467af8f4d2cc0ba44 with a before/after comparison.